calc/states_for_test.go

```diff
@@ -906,6 +906,19 @@ var vxlanWithBlock = empty.withKVUpdates(
 	},
 )
 
+// Minimal VXLAN set-up with a MAC address.
+var vxlanWithMAC = empty.withKVUpdates(
```

Comment:
Think you need to `s/empty/vxlanWithBlock/`

Reply:
Acknowledged and changed the code.